engine/swarm/how-swarm-mode-works/services.md

```diff
@@ -68,6 +68,31 @@ schedules tasks to worker nodes.
 
 ![services flow](../images/service-lifecycle.png)
 
+### Pending services
+
+You can even configure a service in such a way that no node currently in the
+swarm can run its tasks. Here are a few examples:
+
+- If all nodes are paused or drained, and you create a service, it will be
+  pending until a node becomes available. In reality, the first node to become
+  available will get all of the tasks, so this is not a good thing to do in a
+  production environment.
+
+- You can reserve a specific amount of memory for a service. If no node in the
+  swarm has the required amount of memory, the service will remain in a pending
+  state until a node is available which can run its tasks. If you specify a very
+  large value, such as 500 GB, the task will be pending forever, unless you
+  really have a node which can satisfy it.
+
+- You can impose placement constraints on the service, and the constraints may
+  not be able to be honored at a given time.
```

Comment:
Reading this again; this section may be read as "instructions for creating a pending service". In some cases this is desirable (e.g. automatically start deploying when a node becomes available), however if this is just to prevent a service from being deployed, I think we should mention that scaling the service to `0` is the most logical way (or refer to the section about scaling).